language/move-lang/src/cfgir/liveness/mod.rs

```diff
@@ -96,7 +95,8 @@ fn analyze(
     let reverse = &mut ReverseBlockCFG::new(cfg, infinite_loop_starts);
     let initial_state = LivenessState::initial();
     let mut liveness = Liveness::new(reverse);
-    let final_invariants = liveness.analyze_function(reverse, initial_state).unwrap();
+    let (final_invariants, errors) = liveness.analyze_function(reverse, initial_state);
+    assert!(errors.is_empty());
```

Comment:
Could you add a message/ICE on this assert?

Reply:
I don't think it's necessary. The analysis literally doesn't have an errors field in its context. It's mostly here to catch it if someone adds them